qml/pages/AboutPage.qml

```diff
@@ -0,0 +1,160 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+
+import '../components'
+
+Page {
+    id: aboutpage
+
+    readonly property string _APP_VERSION: appVersion
+    readonly property string _APP_BUILD_NUMBER: appBuildNum
+
+    SilicaFlickable {
+        anchors.fill: parent
+        contentHeight: column.height
+
+        Column {
+            id: column
+            width: parent.width
+
+            PageHeader {
+                title: qsTr("About")
+            }
+
+            Image {
+                id: icon
+                source: "image://theme/harbour-nayttamo"
+                anchors.left: parent.left
+                anchors.leftMargin: (aboutpage.width - icon.width) / 2
+            }
+
+            HorizontalSeparator { }
+
+            Label {
+                id: appname
+                text: "Näyttämö"
+                anchors.left: parent.left
+                anchors.leftMargin: (aboutpage.width - appname.width) / 2
+                color: Theme.primaryColor
+            }
+
+            Label {
+                id: version
+                text: qsTr("Version %1.%2").arg(_APP_VERSION).arg(_APP_BUILD_NUMBER)
+                anchors.left: parent.left
+                anchors.leftMargin: (aboutpage.width - version.width) / 2
+                color: Theme.primaryColor
+                font.pixelSize: Theme.fontSizeExtraSmall
+            }
+
+            HorizontalSeparator { }
+
+            SectionHeader {
+                text: qsTr("General")
+            }
+
+            TextArea {
+                width: aboutpage.width
+                readOnly: true
+                color: Theme.primaryColor
+                font.pixelSize: Theme.fontSizeSmall
+                text: qsTr("Native YLE Areena for Sailfish OS")
+            }
+
+            HorizontalSeparator { }
+
+            SectionHeader {
+                text: qsTr("Contributors")
+            }
+
+            Repeater {
+
+                model: ["joonne", "mlehtima"]
+
+                TextField {
```

Comment:
Maybe Label would be simpler for these? Also could help with the too large space between the author names.

Reply:
True, it actually has been a bit wonky looking in my other app also :grin: